components/ephemeral_storage/browser/ephemeral_storage_tab_helper.cc

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ephemeral_storage/browser/ephemeral_storage_tab_helper.h"
+
+#include <string>
```

Comment:
add a space below, system headers should be grouped separately

Reply:
Done.